src/rust/plugin-registry/src/static_files/hax_docker_analyzer.nomad

```diff
@@ -0,0 +1,284 @@
+        RUST_BACKTRACE = 1
+      }
+    }
+
+    # A level of indirection in front of the Graph Query Service that prevents
+    # the plugin from querying other tenants' data.
+
+    task "graph-query-proxy-sidecar" {
+      driver = "docker"
+
+      config {
+        image = var.graph_query_proxy_image
+      }
+
+      template {
+        data        = var.observability_env_vars
+        destination = "observability.env"
+        env         = true
+      }
+
+      env {
+        GRAPH_QUERY_PROXY_BIND_ADDRESS = "0.0.0.0:${NOMAD_PORT_graph-query-proxy}"
+        GRAPH_QUERY_CLIENT_ADDRESS     = "http://${NOMAD_UPSTREAM_ADDR_graph-query}"
+        TENANT_ID                      = var.tenant_id
+
+        RUST_LOG       = var.rust_log
+        RUST_BACKTRACE = 1
+      }
+
+      resources {
+        cpu    = 25  // MHz
+        memory = 128 // MB
+      }
+    }
+
+    restart {
+      attempts = 1
+      delay    = "5s"
+    }
+  }
+
+  group "plugin" {
+    network {
+      mode = "bridge"
+      // TODO
+      // dns {
+      //   servers = local.dns_servers
+      // }
+      port "plugin" {}
+    }
+
+    count = var.plugin_count
+
+    service {
+      name = "plugin-${var.plugin_id}"
+      port = "plugin"
+      tags = [
+        "plugin",
+        "tenant-${var.tenant_id}",
+        "plugin-${var.plugin_id}"
+      ]
+
+      connect {
+        sidecar_service {
+        }
+      }
+
+      check {
+        type     = "grpc"
+        port     = "plugin"
+        interval = "10s"
+        timeout  = "3s"
+      }
+    }
+
+    # a Docker task holding:
+    # - the plugin binary itself (mounted)
+    task "plugin" {
+      driver = "docker"
+
+      config {
+        ports = ["plugin"]
+
+        image      = var.plugin_runtime_image
+        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
+        command = trimspace(<<EOF
+        chmod +x "${PLUGIN_BIN}"
+        "${PLUGIN_BIN}"
+EOF
+        )
+
+        mount {
+          type   = "bind"
+          target = "/mnt/nomad_task_dir"
+          source = "local/"
+          # sigh - we need to `chmod +x` the binary, hence, mount is rw
+          readonly = false
+        }
+      }
+
+      artifact {
+        source      = var.plugin_artifact_url
+        destination = "local/plugin.bin"
+        mode        = "file"
+        headers {
+          x-amz-expected-bucket-owner = var.aws_account_id
+          x-amz-meta-client-id        = "nomad-deployer"
+        }
+      }
+
+      template {
+        data        = var.observability_env_vars
+        destination = "observability.env"
+        env         = true
+      }
+
+      env {
+        TENANT_ID  = "${var.tenant_id}"
+        PLUGIN_ID  = "${var.plugin_id}"
+        PLUGIN_BIN = "/mnt/nomad_task_dir/plugin.bin"
+        # Consumed by GeneratorServiceConfig
+        PLUGIN_BIND_ADDRESS = "0.0.0.0:${NOMAD_PORT_plugin}"
+
+        # Should we make these eventually customizable?
```

Comment:
What does "eventually customizable" mean? And, should we?

Reply:
i'll remove this comment. again: pre-existing